languages/de.ini

```diff
@@ -906,6 +906,10 @@ page_last = "Zur letzten Seite springen"
 page_next = "Zur nächsten Seite"
 page_num = "Seite %%page%%"
 page_prev = "Zur vorherigen Seite"
+page_reload_on_select_hint = "Page will refresh when filter '%%filter_name%%' is selected"
```

Comment:
I notice that the German translations are in English for some reason. I don't think it's worth changing these until we come up with our final design, but just wanted to highlight the issue so we don't forget!

Reply:
Since all of the facet selectors are links, I think it is expected that navigation will occur after activation (reloading in this case).